### ISSUE TITLE
[FEATURE] Add simple guidance to your "[BUG]" issue template to highlight basic limitations

### ISSUE THREAD

**Issue by nmstoker** [pyscript/pyscript#382]
**Is your feature request related to a problem? Please describe.**
PyScript is creating a lot of enthusiasm (rightly so 🎉 ) but many people seem to be missing the fundamental limitations with it and they try naive things, such as attempting things that could never work within the browser sandbox.

They then raise issues when they perceive this to be a "bug" or they ask about it on the [PyScript community boards](https://community.anaconda.cloud/c/tech-topics/pyscript/41), even though these queries ought to be pretty much completely avoidable.

**Describe the solution you'd like**
To help deter needless issues or encourage more informed ones at least, it would be helpful to briefly highlight basic limitations of PyScript (ie things that are clearly going to be impossible) in the repo's [[BUG] Issue Template](https://github.com/pyscript/pyscript/blob/main/.github/ISSUE_TEMPLATE/bug_report.md).

Including a link to a page covering limitations + work-arounds in more detail would give flexibility (so you didn't need to update the template too often).

**Describe alternatives you've considered**
Highlighting the limitations at the various "entry points" that people get to the repo / examples pages could help. This is done to an extent already but clearly a decent number of people are missing it and ploughing on unaware.

Such cases will never be _completely_ avoided but doing what can be done to help people get the right mental model of how it works will prepare them to avoid disappointment and reduce noise with needless issues 🙂 

Also it's important not to "shutdown" discussion on such topics - often there's a legitimate need behind the naive approaches being taken, and deecnt alternatives are often likely to exist, it's just more helpful if people get to that conversation already aware of the limitations rather than being under the illusion that what they did ought to work.

**Additional context**
Examples of people not realising what cannot be done:

- https://github.com/pyscript/pyscript/issues/295
- https://github.com/pyscript/pyscript/issues/237
- https://github.com/pyscript/pyscript/issues/162
- https://github.com/pyscript/pyscript/issues/377
- https://community.anaconda.cloud/t/unable-to-connect-to-pymongo/20635
- https://community.anaconda.cloud/t/the-requests-module/20494

**Comment by marimeireles** [comment]
I think this is a great idea, would you be available to work on something like this?
@fpliger and @antocuni have an opinion on this? :)

**Comment by philippjfr** [comment]
Strongly agree, the most common limitations should be highlighted (i.e. networking stack issues) but we should also point people to the FAQ and ask them to search existing issues before filing theirs. In addition I usually also like to point people to resources that cover how to file a good bug report, such as Matt Rocklin's excellent blog post: https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports

**Comment by nmstoker** [comment]
@marimeireles happy to have a go at coming up with some initial text. I like @philippjfr 's points and would be good to have input from others. 

Also i totally understand if someone from the Anaconda side prefers to handle this (eg to get the nuance spot on / aligned with your plans)

**Comment by marimeireles** [comment]
@nmstoker we'd be happy if you'd like to help =)
After I posted this on the chat, a colleague pointed me to https://github.com/pyscript/pyscript/pull/264
Are these mapping nicely to what you had in mind? Please feel free to chime in or open a new PR if you'd like to add something.

**Comment by antocuni** [comment]
I think it's a very good idea.
I also agree with Mariana that #264 is the way to go, so probably we should just improve it (or merge it and then improve it later)

**Comment by nmstoker** [comment]
Sounds like it'll be clearer after #264 is merged.
That will help as it will separate out distinct kinds of issues but from looking at the file changes, if I'm reading it correctly, it doesn't appear to mention limitations, so there could well be scope for adding something at that point.

**Comment by marimeireles** [comment]
Seems like this was resolved.
Thanks for the input @nmstoker!
